fsm_test.go

```
@@ -0,0 +1,231 @@
+
+// fsmCase is one row of the machine's transition table. outcome is the effect's
+// own recorded decision (the Verdict for a review, empty otherwise); actor is
+// the Flow performing the effect ("" when the row only pins the table).
+type fsmCase struct {
+	from    deliveryState
+	effect  effect
+	facts   subjectFacts
+	outcome string
+	actor   string
+	to      deliveryState
+	wantErr bool
```

golangci-lint (2.12.2)

[error] field wantErr is unused

(unused)